flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java

```diff
@@ -132,9 +132,9 @@
 
 
 
-	private final Configuration config;
+	private Configuration config;
```

Comment:
> Could we make the configDir parameter a general parameter that goes before run, list, cancel, etc.?
Sounds good. `configDir` must be positional after `flink` or I may do something like this `flink run --configDir /my/custom/dir -p 10 /path/to/jar`?

> You could parse the config parameter in the constructor and initialize the config only once. Then continue with the normal parsing.
Now `configDir` is traversal on all actions, i can do it.